modules/lm-coursier/src/main/contraband/publication.json

```diff
@@ -9,19 +9,29 @@
       "fields": [
         {
           "name": "name",
-          "type": "String"
+          "type": "String",
+          "since": "0.0.0"
         },
         {
           "name": "type",
-          "type": "Type"
+          "type": "Type",
+          "since": "0.0.0"
         },
         {
           "name": "ext",
-          "type": "Extension"
+          "type": "Extension",
+          "since": "0.0.0"
         },
         {
           "name": "classifier",
-          "type": "Classifier"
+          "type": "Classifier",
+          "since": "0.0.0"
+        },
+        {
+          "name": "attributes",
```

Comment:
Why is that extra field needed? An `Attributes` instance can be built from the `type` and `classifier` field (like you did for the default value).

Reply:
I thought I saw it being part of `Publication`. I'd fix this to make it same as coursier/coursier.

Reply:
Addressed in 6355338735046930e5af448ef3f3a976e98c548d